src/logic/safe/api/__tests__/fetchTokenCurrenciesBalances.test.ts

```diff
@@ -2,50 +2,53 @@ import axios from 'axios'
 
```

Comment:
This test was out-of-date. Essentially it was broken but nobody noticed it. Is it useful at all? Is there a way to handle such cases?